dpc-api/src/test/java/gov/cms/dpc/api/APITestHelpers.java

```diff
@@ -199,4 +198,33 @@ private static Validator provideValidator(InjectingConstraintValidatorFactory fa
                 .configure().constraintValidatorFactory(factory)
                 .buildValidatorFactory().getValidator();
     }
+
+    public static Practitioner createPractitionerResource(String NPI, String orgID) {
```

Comment:
```suggestion
    public static Practitioner createPractitionerResource(String npi, String orgID) {
```
Would you be okay with making this and the MBI parameter in `createPatientResource()` lowercase? Might get mistaken for constants if someone is scanning the code quickly.